src/Components/Artwork/GridItem.tsx

```diff
@@ -89,6 +94,26 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
     }
   }
 
+  renderBidLabel() {
+    return (
+      <Badge
+        style={{
+          position: "absolute",
+          bottom: "8px",
+          left: "8px",
+          borderRadius: "2px",
+          letterSpacing: "0.3px",
+          padding: "2px 5px 0px 5px",
+          backgroundColor: "white",
+          boxShadow: "0 2px 5px 0 rgba(0, 0, 0, 0.1)",
+          textTransform: "uppercase",
```

Comment:
no reason, i was just fiddling with it. When I add more badges, I may wind up refactoring in to a `BadgesContainer`, etc. (since an artwork can have multiple badges and they should be stacked next to each other).

Reply:
I see 👍